Tests/Rules/AvoidUnusableParameter.tests.ps1

```diff
@@ -0,0 +1,33 @@
+﻿$ruleName = 'PSAvoidUnusableParameter'
+
+Describe "AvoidUnusableParameter" {
+    Context "When there are violations" {
+
+        # It "returns violations when the parameter name is 1 defined in function header" {
```

Comment:
I assume this is commented out as it is WIP

Reply:
Yes, the test I added is handling function parameters defined in parameters block, like:
`function p2 {param([switch]$2) if ($2) {"Yes"} else {"No"}}`
If function is using short notation, then my rule is not catching it.
`function p1 ([switch]$1) {if ($1) {"Yes"} else {"No"}}`
Based on AST analysis, I need to parse `EndBlock`, but I need to review details.